docs/nlp/fine_tune_bert.ipynb

```diff
@@ -1189,7 +1189,7 @@
         "id": "qnxysGUfIgFQ"
       },
       "source": [
-        "Create an instance of this export-model and save it:"
+        "Create an instance of this export model and save it:"
```

Comment:
Keeping dash here makes more sense I think. Or we can make it "exported model".

Reply:
Sure! LGTM. Thank you!